src/App.js

```diff
@@ -1,22 +1,33 @@
 import Pagination from "./common/Pagination";
 import Section from "./common/Section";
-import MovieList from "./feautures/MovieList";
+import MovieList from "./features/MovieList";
+import ActorTile from "./features/actors/ActorTile";
 import Header from "./common/Header";
-import { HashRouter } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
+
 
 function App() {
   return (
-    <HashRouter>
+    <>
       <Header />
-      <Section
-        title="Popular movies"
-        body={
-          <MovieList />
-        }
-      />   
+      <Routes>
```

Comment:
When we open our page we should be redirected to movies page.
To do that we can make a route `<Route path="/" element={<Navigate to="/movies" />} />` 😉